src/main/java/com/glodblock/github/client/gui/GuiFCImgButton.java

```diff
@@ -66,6 +67,14 @@ public GuiFCImgButton(final int x, final int y, final String idx, final String v
             this.registerApp(22, "SWITCH", "DISABLE", "disable");
             this.registerApp(23, "CONFIG", "YES", "open_configuration");
             this.registerApp(24, "HIGHLIGHT", "YES", "block_highlight");
+            this.registerApp(30, "MAGNET_CARD", "INV", "magnet_card.inv");
+            this.registerApp(30, "MAGNET_CARD", "ME", "magnet_card.me");
+            this.registerApp(30, "MAGNET_CARD", "FILTER", "magnet_card.filter");
+            this.registerApp(31, "MAGNET_CARD", "OFF", "magnet_card.off");
+            this.registerApp(33, "ENABLE_12x", "ENABLE", "enable");
+            this.registerApp(34, "DISABLE_12x", "DISABLE", "disable");
+            this.registerApp(36, "RESTOCK", "ENABLE", "restock.on");
+            this.registerApp(37, "RESTOCK", "DISABLE", "restock.off");
```

Comment:
Should these be consecutive numbers?
```suggestion
            this.registerApp(31, "MAGNET_CARD", "OFF", "magnet_card.off");
            this.registerApp(32, "ENABLE_12x", "ENABLE", "enable");
            this.registerApp(33, "DISABLE_12x", "DISABLE", "disable");
            this.registerApp(34, "RESTOCK", "ENABLE", "restock.on");
            this.registerApp(35, "RESTOCK", "DISABLE", "restock.off");
```